Header        header
bool          is_valid
ros_leap/Hand[] hands